### ISSUE TITLE
If destination directory doesn't exist, Copy-FileToNavContainer fails

### ISSUE THREAD

**Issue by ShadowXVII** [microsoft/navcontainerhelper#971]
The `Move-Item` command doesn't create the destination directory first if it doesn't exist. Unlike `Copy-Item -Force` which creates directories, `Move-Item -Force` doesn't seem to and instead fails the command.

> If the ... directory (or any other element of the destination path) does not exist, the command fails. The missing directory is not created for you, even if you use the Force parameter. ([source](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.management/move-item?view=powershell-7#examples))

https://github.com/microsoft/navcontainerhelper/blob/4bf885d513a1bdaf59af2a0113cee3ad428f9c21/Misc/Copy-FileToNavContainer.ps1#L40

Perhaps it could be preceded by something like this? `New-Item -Force` does create subdirectories if they don't exist.

```
if( -Not (Test-Path $containerPath))
{
    New-Item -Path $containerPath -ItemType Directory -Force | Out-Null
}
```

I did this by putting a `Invoke-ScriptInNavContainer` with the above beforehand, but it just seems more neat if the Copy could handle it.

**Comment by freddydk** [comment]
For the ease of using the function, I agree with you.

**Comment by freddydk** [comment]
Shipped in 0.6.5.7
